reframe/frontend/printer.py

```diff
@@ -42,16 +46,22 @@ def status(self, status, message='', just=None, level=logging.INFO):
         else:
             status = status.ljust(self.status_width - 2)
 
+        status_stripped = status.strip().lower()
         if self.colorize:
-            status_stripped = status.strip().lower()
             if status_stripped == 'skip':
                 status = color.colorize(status, color.YELLOW)
             elif status_stripped in ['fail', 'failed', 'error']:
                 status = color.colorize(status, color.RED)
             else:
                 status = color.colorize(status, color.GREEN)
 
-        logging.getlogger().log(level, '[ %s ] %s' % (status, message))
+        final_msg = f'[ {status} ] '
+        if status_stripped in ['ok', 'skip', 'fail', 'error']:
+            self._progress_count += 1
+            final_msg += f'({self._progress_count}/{self._total_count}) '
```

Comment:
I'd be nicer here if you padded the progress count such that the output is like this:

```
(  9/100)
( 90/100)
(100/100)
```

Reply:
Yes I was thinking of padding but the question is do we like padding in both policies?
This looks nicer:
```
[       OK ] ( 8/10) T3 on generic:login using builtin-gcc
[     FAIL ] ( 9/10) T2 on generic:login using builtin-gcc
[     FAIL ] (10/10) T7
```
But I wasn't sure about the serial:
```
[----------] started processing T7 (T7)
[ RUN      ] T7 on generic:login using builtin-gcc
[     FAIL ] ( 8/10) T7
[----------] finished processing T7 (T7)

[----------] started processing T8 (T8)
[ RUN      ] T8 on generic:login using builtin-gcc
[     FAIL ] ( 9/10) T8 on generic:login using builtin-gcc
[----------] finished processing T8 (T8)

[----------] started processing T9 (T9)
[ RUN      ] T9 on generic:login using builtin-gcc
[     FAIL ] (10/10) T9
[----------] finished processing T9 (T9)
```
I will keep the padding in both places to not overcomplicate the code.

Reply:
> I will keep the padding in both places to not overcomplicate the code.

Yep, both places.